src/coreclr-debug/activate.ts

```diff
@@ -155,9 +157,16 @@ export class DebugAdapterExecutableFactory implements vscode.DebugAdapterDescrip
 
         // debugger has finished installation, kick off our debugger process
 
+        // Check for targetArchitecture
+        let targetArchitecture = "";
+        if (this.platformInfo.isMacOS())
+        {
+            targetArchitecture = _session?.configuration?.targetArchitecture ?? this.platformInfo.architecture;
```

Comment:
>targetArchitecture [](start = 12, length = 18)

Should we fail if target architecture is invalid (maybe just check if the vsdbg-ui exists to validate?)